### PR TITLE
Add support for the Reporting resources

### PR DESCRIPTION
**Pull request by remi-stripe**
Adding support for the new Reporting resources. Those are in OpenAPI but not yet public but since they are starting to be used and are stable we should merge them.

It's WIP for now while I get some feedback on the implementation and the shape of the resources.

cc @stripe/api-libraries
cc @brahn-stripe

### CODE REVIEW

File: stripe/api_resources/reporting/report_type.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+from __future__ import absolute_import, division, print_function
+
+from stripe.api_resources.abstract import CreateableAPIResource
+from stripe.api_resources.abstract import ListableAPIResource
+
+
+class ReportType(CreateableAPIResource, ListableAPIResource):
```
Comment (ob-stripe) [inline stripe/api_resources/reporting/report_type.py]: This resource is not creatable.

**Comment by remi-stripe** [comment]
Fixed PTAL
r? @ob-stripe

**Comment by ob-stripe** [comment]
Released as 2.7.0.

**Pull request merged**